Tests/LibJS/test262-runner.cpp

```diff
@@ -654,14 +663,25 @@ int main(int argc, char** argv)
 #define DISARM_TIMER() \
     alarm(0)
 
-    auto standard_input = Core::File::standard_input();
+    auto standard_input_or_error = Core::Stream::File::standard_input();
+    if (standard_input_or_error.is_error())
+        return 7;
```

Comment:
Can we use a less magic number?
Same with the three below

(It's always seven or three......)

Reply:
Sure, it's just to indicate the type of failure in case you're running it yourself.
Added a bunch of constants.